xclim/data/fr.json

```diff
@@ -1069,7 +1069,20 @@
     "long_name": "Jours avec forte accumulation de neige",
     "description": "Nombre de jours où l'accumulation de neige est au-dessus ou égale à {thresh}.",
     "title": "Jours avec accumulation de neige supérieure à un seuil",
-    "abstract": "Nombre de jours où l'accumulation de neige est au-dessus ou égale à un seuil donné."
+    "abstract": "Nombre de jours où l'accumulation de neige est au-dessus ou égale à un seuil donné.",
+    "_deprecated_version": "0.47.0"
```

Comment:
Woops. When I said the next version, I meant the next release (v0.46.0). We'll be free to remove it in v0.47.0. My bad!

Reply:
Wasn't sure about it, so "_deprecated_version" means the last version where this will be in the code, got it.

Reply:
Deprecated here is more like "the first version where we knew this function wasn't going to stick around. Use it production at your own risk." haha